config/config.example.yml

```diff
@@ -13,6 +13,8 @@ ui:
   rateLimiter:
     windowMs: 60000 # 1 minute
     max: 500 # limit each IP to 500 requests per windowMs
+  # Show the file access status in items lists
+  showAccessStatuses: false
```

Comment:
I agree, but it says `# Item Page Config` on top if this section. Since it's not related to the item's page at all, should we rename it to something more generic (like Item Configuration)? The same problem occurs in the file `item-page-config.interface.ts`. I don't know, it's a bit confusing to me.

Reply:
@nibou230 : Sure, feel free to rename the comments if they are inaccurate now. It seems fine to change this to say `# Item Configuration`.  Obviously, as we add more code, sometimes these minor refactors are necessary for things to make "sense". So, please feel free to suggest/make such minor changes in your PRs & just let us know why you chose to do so (in the PR description or similar)